controllers/apply/awards-for-all/transforms.js

```diff
@@ -39,11 +38,10 @@ function transformOrgHasDifferentTradingName(applicationData) {
 
 function transform(applicationData) {
     // Add any default transforms here
-    let transformsToRun = [transformOrgHasDifferentTradingName];
-
-    if (config.get('awardsForAll.enableNewDateRange') === true) {
-        transformsToRun.push(transformProjectDateRange);
-    }
+    const transformsToRun = [
+        transformOrgHasDifferentTradingName,
+        transformProjectDateRange
```

Comment:
I wonder if we should add notes to these (or even calendar reminders) set 3 months after they went live, so we can delete them? There'll come a point when all the pending applications with the old format have been deleted, and any submitted ones will have either used the new format, or were submitted before this change (and therefore shouldn't be affected)?

Reply:
Sent a calendar invite for three months time. We log usage so we can confirm they are no longer used and then delete.